webapp/channels/src/components/invitation_modal/invite_view.tsx

```diff
@@ -103,7 +103,7 @@ export default function InviteView(props: Props) {
                     defaultMessage: 'team invite link {inviteURL}',
                 }, {inviteURL})
             }
-            className='InviteView__copyLink'
```

Comment:
It looks like there's some CSS that uses `InviteView__copyLink` in `invite_view.scss`. Could that be removed, or is it still needed? There's a comment on it that links it to https://mattermost.atlassian.net/browse/MM-40611, but it's also using `.tertiary-button` which I don't even see in the old version of this code